### ISSUE TITLE
圣保罗地铁模板/São Paulo Metro templates

### ISSUE THREAD

**Issue by linchen1965** [wongchito/RailMapGenerator#212]
[Metro SP.zip](https://github.com/wongchito/RailMapGenerator/files/7788786/Metro.SP.zip)
圣保罗地铁1~5号线以及单轨（15号线）的模板，将来还会继续上传圣保罗都市圈铁路公司的其他七条线路的模板。
Templates of São Paulo Metro Line 1 to Line 5 and The monorail Line 15. The seven Lines (Line 7 to Line 13) of Companhia Paulista de Trens Metropolitanos  will be updated soon.

**Comment by linchen1965** [comment]
[CPTM.zip](https://github.com/wongchito/RailMapGenerator/files/7806499/CPTM.zip)
圣保罗都市圈铁路7~13号线模板制作完成
Templates of seven Lines (Line 7 to Line 13) of CPTM finished

**Comment by thekingofcity** [comment]
Great job. We'll merge them as soon as possible.

**Comment by thekingofcity** [comment]
Hi, preparing the line name according to the [Wiki](https://github.com/wongchito/RailMapGenerator/wiki/How-to-add-color-palette-and-line-templates#line-templates) will greatly speed up the merging process.

```
shmetro: [
    {
        filename: 'sh1',
        name: {
            en: 'Line 1',
            'zh-Hans': '1号线',
            'zh-Hant': '1號線',
        },
    },
    {
        filename: 'sh2',
        name: {
            en: 'Line 2',
            'zh-Hans': '2号线',
            'zh-Hant': '2號線',
        },
    },
    ...
],
```

**Comment by linchen1965** [comment]
It's better to check this txt again before using. I'm not sure about the codes...
[São Paulo metropolitan transport .txt](https://github.com/wongchito/RailMapGenerator/files/7829341/Sao.Paulo.metropolitan.transport.txt)

**Comment by thekingofcity** [comment]
> It's better to check this txt again before using. I'm not sure about the codes... [São Paulo metropolitan transport .txt](https://github.com/wongchito/RailMapGenerator/files/7829341/Sao.Paulo.metropolitan.transport.txt)

Hi, the line name should have a Portuguese name. 

```
    saopaulometro: [
        {
            filename: 'sp1',
            name: {
                en: 'Line 1',
                pt: 'Linha 1',
                'zh-Hans': '1号线',
                'zh-Hant': '1號線',
            },
        },
        {
            filename: 'sp2',
            name: {
                en: 'Line 2',
                pt: 'Linha 2',
                'zh-Hans': '2号线',
                'zh-Hant': '2號線',
            },
        },
```

But don't worry, I added it for you.

**Comment by thekingofcity** [comment]
Hi, they are added in 95a1913c and released in 3.10.32

Also, be sure to check out the lasted Wiki for the updated instruction on how to add line templates if you wish to add another one.
